src/components/connectAdvanced.js

```diff
@@ -23,6 +23,127 @@ function storeStateUpdatesReducer(state, action) {
   return [action.payload, updateCount + 1]
 }
 
+function useIsomorphicLayoutEffectWithArgs(create, deps, args) {
```

Comment:
Can we swap the last two args and use slightly more descriptive naming?

```suggestion
function useIsomorphicLayoutEffectWithArgs(effectFunc, effectArgs, dependencies) {
```

The reason being that it reads more naturally to say that you're "calling this function with these args" as a contiguous phrase. Also, it's relatively idiomatic React/useEffect to have the dependency array at the end of the function call.